cypress/e2e/article.cy.js

```diff
@@ -1,13 +1,52 @@
-describe('', () => {
-  before(() => {
+describe('Article flow', () => {
+  let user;
+  let articleData;
 
+  beforeEach(() => {
+    cy.visit('');
+    cy.task('generateUser').then((generateUser) => {
+      user = generateUser;
+    });
+    cy.task('generateArticle').then((generateArticle) => {
+      articleData = generateArticle;
+    });
   });
```

Comment:
To make your tests more isolated and independent, it's recommended to avoid sharing state between tests. Right now, you're setting `user` and `articleData` in the `beforeEach` hook, which means these variables will be shared across all tests in this suite. This can lead to problems when tests are run in parallel or in a different order. Instead, you could generate a new user and a new article inside each test.